pina/geometry/difference_domain.py

```diff
@@ -1,28 +1,87 @@
 """Module for Location class."""
-
+import torch
 from .location import Location
+from .exclusion_domain import Exclusion
+from ..utils import check_consistency
 from ..label_tensor import LabelTensor
+import random
 
 
-class Difference(Location):
-    """
-    """
+class Difference(Exclusion):
+    """ PINA implementation of Difference of Domains."""
 
-    def __init__(self, first, second):
+    def __init__(self, geometries):
+        """ PINA implementation of Difference of Domains.
 
-        self.first = first
-        self.second = second
+        :param list geometries: A list of geometries from 'pina.geometry' 
+            such as 'EllipsoidDomain' or 'CartesianDomain'.
 
```

Comment:
This documentation is pretty misleading. I do not understand how difference work. 

A list of geometries from 'pina.geometry'   is vague...

You should explain how is the difference taken